src/stories/5-custom/array-field/array-field.stories.tsx

```diff
@@ -167,24 +174,27 @@ const SCHEMA_NESTED_ARRAY: Record<string, TFrontendEngineFieldSchema> = {
 				fieldSchema: {
 					grid: {
 						uiType: "grid",
-						style: { marginTop: 16, marginBottom: 16 },
+						style: { marginTop: 16, marginBottom: 16, border: "1px solid black", padding: 16 },
```

Comment:
the styles here and from line 144 are not necessary

instead, you can create a new div and wrap it around `testArray`

```
wrapper: {
	uiType: "div",
	style: {
		padding: "1rem",
		borderRadius: 4,
		border: `1px solid #E0E4E5`,
	},
	columns: { mobile: 4, tablet: 8, desktop: 12 },
	children: {
		testArray: {
			// ...
		}
	}
}
```